Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "user"
+version = "0.1.0"
+edition = "2021"
+authors = ["Chris Berner <christopherberner@gmail.com>"]
```

Comment:
The authors should be:
* Christopher Berner <me@cberner.com>
* Casey Rodarmor <casey@rodarmor.com>